### PR TITLE
Display the impact per kg of product

### PR DESCRIPTION
**Pull request by magopian**
cf [notion](https://www.notion.so/fd0068a7db8348c584cd53a3eea399f2?v=90c1a9d44ff845c090b6f491065ba2ef&p=13b41d53b0354d1ea5cab648107f7ad2&pm=s)

### CODE REVIEW

File: src/Page/Food/Builder.elm [hunk 1]
```diff
@@ -773,14 +773,29 @@ sidebarView session db model results =
...
+                    , h3 [ class "h6 m-0 mt-2" ] [ text <| "Impact pour " ++ totalWeightStr ++ "kg de produit" ]
+                    , div [ class "display-5 lh-1" ]
                         [ results.total
                             |> Format.formatFoodSelectedImpact model.impact
                         ]
-                    , small [ class "d-flex align-items-center gap-1" ]
+                    , small []
                         [ Icon.warning
-                        , text "Attention, ce résultat est partiel"
+                        , text " Attention, ce résultat est partiel"
```
Comment (n1k0) [inline src/Page/Food/Builder.elm:798]: This should now read "Attention, ces résultats sont partiels"

File: src/Views/Format.elm [hunk 2]
```diff
@@ -54,6 +55,14 @@ formatFoodSelectedImpact { trigram, unit, decimals } =
         >> formatRichFloat decimals unit
 
 
+formatFoodSelectedImpactPerKg : Impact.Definition -> Mass -> Impacts -> Html msg
+formatFoodSelectedImpactPerKg { trigram, unit, decimals } mass =
+    Impact.getImpact trigram
+        >> Unit.impactToFloat
+        >> (\impact -> impact / Mass.inKilograms mass)
+        >> formatRichFloat decimals unit
```
Comment (n1k0) [inline src/Views/Format.elm:63]: So this is just and only an Html view thing? Don't we ever expect to have to expose this unitin API results? 

Shouldn't we rather have a dedicated function in the `Unit` module?

Comment (magopian) [reply src/Views/Format.elm:63]: There's at the moment nothing in `Unit` related to `Impacts` or `Impact.Definition` or `Html`, so I'm not sure what you're suggesting? This helper is the pendant of `formatFoodSelectedImpact` defined just above.

Comment (n1k0) [reply src/Views/Format.elm:63]: https://github.com/MTES-MCT/ecobalyse/blob/master/src/Data/Unit.elm#L393-L440 ?

Comment (magopian) [reply src/Views/Format.elm:63]: This isn't the same `Impact.Definition` we're talking about, is it? I'm confused

Comment (n1k0) [reply src/Views/Format.elm:63]: I think we should use `formatFoodSelectedImpact` and do the "per kg" conversion before using it. That with or without introducing some helper in `Unit`, like we did for `impactAggregateScore` for example.

Comment (n1k0) [reply src/Views/Format.elm:63]: bah scratch it

**Review by n1k0 — APPROVED** [review]
LGTM ship it

**Pull request merged**